packages/auth/seeds/development/03_access_tokens.js

```diff
@@ -1,40 +0,0 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Should we also remove `database_docker-compose.yml`, along side the `"Shoe Shop" Demo` section in the `mock-account-provider/README.md` ? I don't think it's necessary (like these)